src/VelocityUKF.cpp

```diff
@@ -3,7 +3,7 @@
 using namespace uwv_kalman_filters;
 
 const std::string VelocityUKF::acceleration_measurement = "acceleration";
```

Comment:
> An enum doesn't work since the type cannot know of all the sorts of measurements the users want to have. 

From which point of view ?
 - the base filter class ?
 - the UKF filter class ?
 - the orogen interface ?
 - ... ?